### DISCUSSION TITLE
Firmware update 10.12.114 appears to break ONVIF ContinuousMove

### DISCUSSION THREAD

**Discussion by xtrdustin** [AxisCommunications/discussions#88] [General] [answered]
We're having some trouble with the ContinousMove command, which seems to be caused by the firmware upgrade:

- We were previously on version 9.80.2.2, and sending ONVIF ContinousMove commands, which were working.
- When we upgraded to version "10.12.114 - AXIS OS LTS 2022", we started getting error 400 in response to the ContinuousMove commands.
- When we reverted back to the old version, we were no longer seeing the errors. 

Was there a change in what the ContinuousMove command accepts, and/or something else we need to do for it to work on the new version?

The camera is an AXIS Q6215-LE PTZ Network Camera.

Thanks in advance!

**Comment by xtrdustin** [comment]
Did another test to verify that it wasn't just the restart after reverting that fixed it. Upgraded again to 10.12.114. ContinuousMove commands caused errors. Restarted. Still errors. Reverted again to 9.80.2.2: ContinuousMove returned no errors.

**Comment by mattias-kindborg-at-work** [comment]
Hi @xtrdustin. We are having trouble reproducing this. Could you please provide us with a complete sample we can run to showcase the issue?

**Reply by xtrdustin** [reply]
I'll try upgrading again to the new version on monday and get a capture. Is there any way I can send it to you without making it public?

By the way, we've seen an issue (currently on the 9.x firmware) where GetStatus starts failing with error 400 and the camera has to be restarted. Is that something you've seen that might be fixed in 10.x?  If not, we'll try and get a capture of this as well if it happens again.

**Reply by xtrdustin** [reply]
@mattias-kindborg-at-work I have a couple packet captures. Is there a way I can send them to you without making them public?

**Reply by mattias-kindborg-at-work** [reply]
Add them to a privat GitHub repository and invite me with read privileges.

**Reply by xtrdustin** [reply]
Okay, I added it to https://github.com/xtrdustin/axis-uploads/

**Reply by mattias-kindborg-at-work** [reply]
It seems that current implementation do not support parsing of `xs:durations` with years and months specified (for example `P0Y0M0DT0H0M1.000S`), prefer to specify the timeout as `PT60S` instead.

**Reply by xtrdustin** [reply, marked as answer]
It turns out we don't really need the duration parameter, and were able to upgrade to firmware 11.1.65.

However, this caused our "metadata" media profile to be deleted, and we had to re-create it.
